cfgov/v1/jinja2/v1/includes/atoms/tagline.html

```diff
@@ -14,7 +14,8 @@
    ========================================================================== #}
 
 {% macro render( is_large ) %}
-<div class="a-tagline{{ ' a-tagline__large' if is_large else '' }}">
+<div class="a-tagline{{ ' a-tagline__large' if is_large else '' }}" 
+      aria-label="{{ _('Official website of the United States government') }}">
```

Comment:
Super trivial: Let's align the `class` and `aria-label` … or did your tabbing settings get it to here? Prob best to align it.

Reply:
I just pushed a commit to fix the backend translation piece, which I think fixes this as well.